test/integration/check-ins/check-in.js

```diff
@@ -15,7 +15,7 @@ context('patient check-in', function() {
         return fx;
       })
       .routePatientCheckIn(fx => {
-        fx.data.attributes.finished_at = completedDate.format();
+        fx.data.checkin.finishedTs = completedDate.format();
```

Comment:
so this doesn't looks like it is formatted ISO 8601 and is `YY:MM:DD hh:mm:ss` in the data.  I assume these dates are UTC coming from the server and so I _believe_ we'll need to convert them to utc prior to displaying them as well by using `moment.utc(this.model.get('finishedTs'))`